src/components/FlawCreateForm.vue

```diff
@@ -73,21 +73,26 @@ const getUnembargoDateTime = () => {
         <div class="row">
           <div class="col-6">
             <!--<div>UUID: {{ flaw.uuid }}</div>-->
-            <div class="input-group mb-2"><span class="input-group-text osim-input-fixwidth">Type: </span><input type="text" class="form-control" v-model="modelValue.type" placeholder="e.g. VULNERABILITY" /></div>
-            <div class="input-group mb-2"><span class="input-group-text osim-input-fixwidth">CVE ID: </span><input type="text" class="form-control" v-model="modelValue.cve_id" placeholder="e.g. CVE-2077-1337" /></div>
+          <div class="input-group mb-2">
+            <span class="input-group-text osim-input-fixwidth">Type: </span>
+            <select class="form-control" v-model="modelValue.type">
+              <option disabled value="">VULNERABILITY</option>
```

Comment:
Refer to the LabelSelect for flawType in IssueEdit.vue - we should programmatically populate the values